docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,1918 @@
+{
```

Comment:
<p>maybe note below that these dimensions can be easily plotted with <code>scanpy.pl.embedding</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>good idea, added a line and link to pl.embedding docs</p>